CHANGES.txt

```diff
@@ -1,4 +1,5 @@
-1.16.1 (July 10, 2024)
```

Comment:
why are you removing `1.16.1`?

Reply:
We were going to release v1.16.1 with only vulnerability fixes, but it wasn't released at the end. So I am updating the version as a minor release now.